sql/sys_vars.cc

```diff
@@ -1607,6 +1607,13 @@ static Sys_var_bool Sys_binlog_rows_query(
     DEFAULT(false), NO_MUTEX_GUARD, NOT_IN_BINLOG,
     ON_CHECK(check_session_admin));
 
+static Sys_var_bool Sys_binlog_ddl_query(
```

Comment:
Whenever we add a new server variable, we will need to rerecord a few main suite testcases like
```
1. all_persisted_variables
2. mysqld--help-notwin
and others
```

Please run the main suite once and check if there any test failures.

Reply:
Also, please update the recent Jenkins link in a comment or in the description.